### PR TITLE
[FIX] adjust CSS to resolve issue #93

### PR DESCRIPTION
**Pull request by mhuyck**
Beautifully simple fix that resolves the issue where it arose: in CSS.
Courtesy of: https://github.com/twbs/bootstrap/issues/1768#issuecomment-46519033

### CODE REVIEW

**Comment by mhuyck** [comment]
Hmm. Where did CircleCI go?

File: interface/src/app/analyze/analyze.js [hunk 1]
```diff
@@ -58,9 +54,7 @@ angular.module('adage.analyze', [
       },
 
       scroll_to_id: function(id) {
-        $log.info('scroll_to_id called with: ' + id);
```
Comment (dongbohu) [inline interface/src/app/analyze/analyze.js]: Is this supposed to be debug line?

Comment (mhuyck) [inline interface/src/app/analyze/analyze.js]: Aha, nice catch. I'll take that out.

Comment (mhuyck) [inline interface/src/app/analyze/analyze.js]: Heh. I *did* take that out! Note that it's got a `-` in that diff view.

Comment (mhuyck) [inline interface/src/app/analyze/analyze.js]: Ah... but I'm noticing that I removed the `$anchorScroll` entirely but it's still getting injected above. I'll fix that too.

Comment (dongbohu) [inline interface/src/app/analyze/analyze.js]: Yeah, I noticed that this line is removed. I just want to understand the exact reason. Thanks.

Comment (mhuyck) [inline interface/src/app/analyze/analyze.js]: Right. It had been left in there from when I was initially building the feature. There is no reason for it to stay... it was just for debugging.

**Comment by mhuyck** [comment]
Thanks for the speedy review, @dongbohu!

I see that CircleCI is back. I'll make sure it's happy with my update removing `$anchorScroll` before merging.

**Pull request merged**